dev/CommonStyles/InteractionTests/CommonStylesTests.cs

```diff
@@ -133,5 +133,20 @@ private void RunDensityTests(string buttonName)
                 Verify.AreEqual(densityTestResult, "Pass", "We expect density test result is Pass");
             }
         }
+
+        [TestMethod]
+        public void RunCompactTests()        
+        {
+            using (var setup = new TestSetupHelper("Compact Tests"))
+            {
+                Log.Comment("Click on RunTest");
+                var button = new Button(FindElement.ByName("RunTest"));
+                button.Invoke();
+                Wait.ForIdle();
+
+                var testResult = new TextBlock(FindElement.ById("CompactTestResult")).GetText();
+                Verify.AreEqual(testResult, "Pass", "We expect compact test result is Pass");
```

Comment:
This "Pass" string should really be enforced by SimpleVerify, not redefined here.

Reply:
FindElement will not goes into control template part, so SimpleVerify is in testapp, and it's not in test case. 
We need a "Pass" to show the test result when you clicked a button.
"Pass" or error message is returned by SimpleVerify

Reply:
But SimpleVerify could offer the "Pass" string as a public const. As is it's an internal detail.

Reply:
SimpleVerify is in testapp project and test case in another project. Also SimpleVerify is only used for density testing, 'Pass' string never be checked in testapp itself. so it would be over design to offer 'Pass' as public const.

Reply:
I know this is a silly little helper class used for only this corner of testing. But I also firmly believe that we can afford to follow good design practices on every bit of code we write. Partially just because having good habits makes it less likely we accidentally fall into bad habits when it _does_ matter. But also because silly little code we think doesn't matter has a nasty habit of turning into foundation... because that's the reality of how software organically evolves.

This isn't a fundamental or high risk bit of code. But there are also a lot of small tweaks you could make to alleviate this feedback that would take you very little time.

Reply:
Or if we can't share code just add a comment about // Must match value defined in SimpleVerify

Reply:
```suggestion
                Verify.AreEqual(testResult, "Pass", "We expect compact test result is Pass"); // "Pass" string matches value used by MUXControlsTestApp.SimpleVerify
```